README.md

```diff
@@ -1,3 +1,23 @@
-# Setup
+# Hosted Link
 
-Create .env.development and .evn.test files in the root of this repo. .env.developmet should each set PGDATABASE to be whatever databases are used in ./db/setup.sql respectively. (currently nc_news for development, and nc_news_test for test)
+Link to hosted server: https://news-server-k1ka.onrender.com/api
+
+# Summary
+
+This project is for setting up the back end of a new-esque server. It uses postgresql to create and manage databases via requests made to the app from express.
```

Comment:
I'd say it's more accurate to say that it's a back end for a news site, because news-esque server doesn't really mean anything 😆

You'd have a backend server for a frontend site  - so be clear about your terminology in that THIS is the server bit 👍